partition/grub_test.go

```diff
@@ -88,9 +89,11 @@ func (s *PartitionTestSuite) TestSetBootVer(c *C) {
 	}
 
 	g := newGrub()
-	err := g.SetBootVar("key", "value")
+	err := g.SetBootVars(map[string]string{
+		"key": "value",
+	})
 	c.Assert(err, IsNil)
 	c.Assert(cmds, DeepEquals, [][]string{
-		{"/usr/bin/grub-editenv", g.(*grub).envFile(), "set", "key=value"},
+		{"/usr/bin/grub-editenv", g.(*grub).envFile(), "set", "key=value "},
```

Comment:
Similarly, this also looks like a problem. We don't want that extra space at the end.